system/jlib/jlog.cpp

```diff
@@ -3185,3 +3191,70 @@ TraceFlags loadTraceFlags(const IPropertyTree *ptree, const std::initializer_lis
     }
     return dft;
 }
+
+void ctxlog_report(const LogMsgCategory & cat, const char * format, ...) 
+{
+    va_list args;
+    va_start(args, format);
+    ctxlog_report_va(cat, unknownJob, NoLogMsgCode, format, args); 
```

Comment:
defaultJobInfo rather than unknownJob?

Reply:
unknownJob is translated to defaultJobInfo at a lower level. I used unknownJob for consistency with other places, but I can change if you want